### PR TITLE
Change from `late` to `late final` to make the code a bit stronger

### PR DESCRIPTION
**Pull request by fzyzcjy**
should not break anything imho

### CODE REVIEW

File: flutter_mobx/lib/src/observer_widget_mixin.dart [hunk 1]
```diff
@@ -70,7 +70,7 @@ mixin ObserverElementMixin on ComponentElement {
 
   @override
   Widget build() {
-    late Widget built;
+    late final Widget built;
```
Comment (pavanpodila) [inline flutter_mobx/lib/src/observer_widget_mixin.dart:73]: Can't be final as the `built` widget is created in every build cycle

**Comment by pavanpodila** [comment]
Can' be late final as the variable changes every build cycle

**Comment by fzyzcjy** [comment]
Well imho it is a *local* variable, not a global one. So this can be final.

**Comment by pavanpodila** [comment]
You are right. Missed that part. Point taken

**Comment by fzyzcjy** [comment]
😄

**Pull request merged**